charts/mlp/templates/postgres-service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if not .Values.postgresql.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
```

Comment:
Just realised the MLP chart does not have a provision for the custom labels to track our internal deployments, like Turing does: https://github.com/gojek/turing/blob/main/infra/charts/turing/templates/_helpers.tpl#L9

Not really in the scope of this PR, but something to address later I guess.